.github/labeler.yml

```diff
@@ -1,2 +1,100 @@
-Proposal:
+# Proposals
+proposal:
 - proposals/text/*
+
+# Topics
+topic:tests:
+- test/**/*
+- test/*
+- conftest.py
+topic:docker:
+- docker/*
+- docker-compose.yml
+topic:CI:
+- .github/*
+- .github/**/*
+topic:DX:
+- CONTRIBUTING.md
+- .pre-commit-config.yaml
+- .gitignore
+topic:dependencies:
+- pyproject.toml
+topic:build/distribution:
+- pyproject.toml
+topic:modeling:
+- haystack/modeling/*
+- haystack/modeling/**/*
+- test/modeling/*
+topic:reader:
+- haystack/nodes/reader/*
+- test/nodes/test_reader.py
+topic:retriever:
+- haystack/nodes/retriever/*
+- test/nodes/test_retriever.py
+topic:pipeline:
+- haystack/pipelines/*
+- test/pipelines/*
+topic:rest_api:
+- rest_api/*
+- rest_api/**/*
+topic:prompt_node:
```

Comment:
We might want to use the topic:llm label for these files too or simply replace topic:prompt_node with topic:llm .
prompt_node was used only once so far: https://github.com/deepset-ai/haystack/pulls?q=is%3Apr+label%3Atopic%3Aprompt_node+
llm was used six times: https://github.com/deepset-ai/haystack/pulls?q=is%3Apr+label%3Atopic%3Allm

Reply:
Good idea, let's do this 👍